daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -1123,18 +1107,27 @@ private[lf] object Speedy {
     kontStack
   }
 
+  private[speedy] sealed abstract class Control
+  object Control {
+    final case class Expression(e: SExpr) extends Control
+    final case class Value(v: SValue) extends Control
+    final case class WeAreUnset() extends Control
```

Comment:
Should this be a case object?

Reply:
probably.
not changing here, as they may not survive a following PR which simplifies the Control further.